src/branch.rs

```diff
@@ -68,7 +69,20 @@ impl BranchList {
     }
 
     pub fn fetch(&mut self) -> Result<()> {
-        let output = git_process(&["branch"])?;
+        let config = CONFIG.get().expect("config wasn't initialised");
+
+        let output = match config.options.sort_branches.as_ref() {
+            Some(sort_value) => {
+                let output = git_process(&["branch", &format!("--sort={sort_value}")])?;
+                if output.status.success() {
+                    output
+                } else {
+                    MiniBuffer::push_command_output(&output);
```

Comment:
this will show the error right when someone opens up `gex`. Also the error message isn't very informative. It just says "unknown field name: {whatever you passed into sort_branches}"